robot/EDA/resources/locators_49.py

```diff
@@ -102,6 +102,7 @@
     },
      "eda_settings_cc": {
         "default_cc_checkbox": "//div[text()='Enable Course Connections']/following-sibling::div/descendant::img",
+        "dropdown_field": "//div[text()='{}']/following-sibling::div/select",
```

Comment:
Since your keyword is in EDA.py, it is better to move this locator to "eda_settings" dict rather than this one which deals specifically with Course Connections page.

Reply:
Moved the xpath to eda_settings dictionary